VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
+| 1.1.1   | 3.2.2       | 3.1.0           | 1.0.11               |
```

Comment:
I haven't updated these because they have a bunch of changes and just wanted to update the README